src/contoroller/category/category.js

```diff
@@ -78,13 +65,13 @@ class CategoryController {
   }
 
   //** OK **//
-  async getSome(req, res) {
-    const someCategories = await this.Category.findAll({
-      where: { type: req.params.type, deletedAt: null },
+  async getAll(req, res) {
+    const allCategories = await this.Category.findAll({
+      where: { userId: req.user.id, type: req.query.type, deletedAt: null },
     });
-    if (!someCategories)
+    if (!allCategories)
```

Comment:
In getAll methods you cannot return 404. You will return a list of categories here, it either has some items in it or it is empty.